manifest.php

```diff
@@ -31,7 +31,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '11.2.0',
+    'version' => '12.0.0',
```

Comment:
Shouldn't start the branch name with `breaking` if you modify the major version?